.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "4.2"
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm install -g selenium-standalone
+  - selenium-standalone install
+  - selenium-standalone start > /dev/null 2>&1 &
```

Comment:
I'm still not satisfied at the current state of this -- I would prefer it to be scripted not exclusively for Travis CI. Unfortunately, I don't have time atm the explore what's going wrong in a Windows VM =(